### PR TITLE
Add: Modal for Instructions CSS

### PR DESCRIPTION
**Pull request by firefinchdev**

### CODE REVIEW

**Comment by abhishek97** [comment]
Also add example markup.

**Comment by firefinchdev** [comment]
@abhishek97 
Done

**Comment by firefinchdev** [comment]
Changes for touch devices done (Player Instructions no more visible for them)

File: sass/styles/elements/elements.modal-instructions.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.footer {
+  label {
+    height: 20px;
+    margin: 3%;
+  }
+}
+@media (max-width: 768px) {
+  #modal-overlays {
+    display: none;
```
Comment (vdvibhu20) [inline sass/styles/elements/elements.modal-instructions.scss]: add this to media query layer(/sass/styles/media-query).

**Pull request merged**